### PR TITLE
Change name to ACME; other updates

### PR DESCRIPTION
**Pull request by rdvelazquez**

### CODE REVIEW

File: src/data/members_info.js [hunk 1]
```diff
@@ -47,19 +47,6 @@ export default {
       "googlescholar": "https://scholar.google.com/citations?user=A2ImcaUAAAAJ&hl=en&authuser=1"
     }
   },
-  "Stephanie": {
```
Comment (spond) [inline src/data/members_info.js]: Can we make a "former" group members section?

Comment (spond) [reply src/data/members_info.js]: This is not something we need to do right away

File: src/app.jsx [hunk 2]
```diff
@@ -10,7 +10,7 @@ class About extends Component {
     return(
 			<div id="about">
 				<h2 className="title">About</h2>
-				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
```
Comment (spond) [inline src/app.jsx]: How about 

>We develop statistical models and computational tools for comparative
analysis of sequence data, especially large and rich data sets from
measurably evolving pathogens. We relish scientific and engineering
challenges involved in creating scalable and useable tools for our own
research and for the use by the broader community. We are omnivorous
in applied research problems and currently work on problems in include
molecular epidemiology of HIV, dynamics of immune repertoires,
pathogenesis and within-host evolution of HIV and SIV, and optimal use
of next generational sequencing data.

File: src/data/members_info.js (hunk 1 above)
Comment (rdvelazquez) [reply src/data/members_info.js]: I created an issue (#11). And will plan to add this week.

File: src/app.jsx (hunk 2 above)
Comment (rdvelazquez) [reply src/app.jsx]: I like it!

**Pull request merged**